test/api_integration/apis/data_views/scripted_fields_crud/get_scripted_field/errors.ts

```diff
@@ -68,13 +68,6 @@ export default function ({ getService }: FtrProviderContext) {
       );
     });
 
-    it('returns 404 error on non-existing scripted field', async () => {
-      const response1 = await supertest.get(
-        `/api/index_patterns/index_pattern/${indexPattern.id}/scripted_field/test`
-      );
-      expect(response1.status).to.be(404);
-    });
-
```

Comment:
Complete duplicate of https://github.com/elastic/kibana/blob/2bed553eaca67b0e68e9f8b7cb164d1162f0937f/test/api_integration/apis/data_views/scripted_fields_crud/get_scripted_field/errors.ts#L51-L57